feat: Support for elastic.co in ElasticsearchSriver and improved docs (#2240)
cube-js/cube.js: packages/cubejs-elasticsearch-driver/README.md

# Cube.js Elasticsearch Database Driver

Elasticsearch driver.

> It has been originally built for OpenDistro version (local and cloud-hosted).
> Later enhanced to work with native Elasticsearch and one hosted on elastic.co.

Reviewer: ```suggestion
Works with OpenDistro, native and elastic.co hosted versions of Elasticsearch.
```
